lib/pavilion/config.py

```diff
@@ -17,11 +17,13 @@
 # Figure out what directories we'll search for configuration files.
 PAV_CONFIG_SEARCH_DIRS = [Path('./').resolve()]
 
-if 'HOME' in os.environ:
-    USER_HOME_PAV = Path(os.environ['HOME'], '.pavilion')
-    PAV_CONFIG_SEARCH_DIRS.append(USER_HOME_PAV)
-else:
-    USER_HOME_PAV = None
+try:
+    USER_HOME_PAV = Path('~/.pavilion').expanduser()
```

Comment:
`AttributeError: 'PosixPath' object has no attribute 'expanduser'`

I think you mean `.home()`?

Reply:
Expand user isn't in 3.4, apparently.